py/picca/delta_extraction/expected_fluxes/continuum_variance_expected_flux.py

```diff
@@ -0,0 +1,267 @@
+"""This module defines the class Dr16ExpectedFlux"""
+import logging
+import multiprocessing
+
+import numpy as np
+from scipy.interpolate import interp1d
+
+from picca.delta_extraction.astronomical_objects.forest import Forest
+from picca.delta_extraction.expected_fluxes.dr16_fixed_eta_varlss_expected_flux import (
+    Dr16FixedEtaVarlssExpectedFlux)
+from picca.delta_extraction.expected_fluxes.dr16_fixed_eta_varlss_expected_flux import (  # pylint: disable=unused-import
+    accepted_options, defaults)
+from picca.delta_extraction.utils import find_bins
+
+VAR_PIPE_MIN = np.log10(1e-5)
+VAR_PIPE_MAX = np.log10(2.)
```

Comment:
What is this used for? How are these numbers chosen? What happens if we have pipeline variance smaller or larger than this? It would be good to at least document it.

Reply:
Do we know what fraction of pixels is rejected by these cuts? If it is a small fraction, we probably can ignore it, but it would be a very good sanity check.

Reply:
Wouldn't the better approach maybe to not ignore those, but instead have those be two additional bins with all larger than max in one and all smaller than min in the other? Potentially not using those later, but having the info for completeness

Reply:
Possibly, to be checked. In any case, I would address that in another PR